src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+        String logo = "         z$$$$P\n" +
+                "        d$$$$\"\n" +
+                "      .$$$$$\"\n" +
+                "     z$$$$$\"\n" +
+                "    z$$$$P\n" +
+                "   d$$$$$$$$$$\"\n" +
+                "  *******$$$\"\n" +
+                "       .$$$\"\n" +
+                "      .$$\"\n" +
+                "     4$P\"\n" +
+                "    z$\"\n" +
+                "   zP\n" +
+                "  z\"\n" +
+                " /    M A V I S'\n" +
+                "^";
+        //Store added tasks to an array
```

Comment:
Pretty obvious by the name of the variable? comment might not be needed

Reply:
Redundant comments will be removed in the next push